ipset.go

```diff
@@ -165,9 +165,18 @@ func (s *IPSetBuilder) Clone() *IPSetBuilder {
 	}
 }
 
+func (s *IPSetBuilder) addError(msg string, args ...interface{}) {
+	se := new(stacktraceErr)
+	n := runtime.Callers(3, se.pcs[:])
```

Comment:
Nit: would appreciate a comment for the "3" magic number.

Reply:
Done.